apps.yml

```diff
@@ -2297,3 +2297,1098 @@ apps:
+    name: Foreman
+    op: auth0
+    url: https://foreman.private.mdc1.mozilla.com/callback
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 5UpSojG2Z4qFxnVS8oa9s3m9KQFpMI5h
+    display: 'false'
+    logo: auth0.png
+    name: signoff.shipit.staging.mozilla-releng.net
+    op: auth0
+    url: https://signoff.shipit.staging.mozilla-releng.net/oidc_callback
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: MmT719PU6y84GsRkym00nVvVHi2y5hPa
+    display: 'false'
+    logo: auth0.png
+    name: testwebapp.private.mdc1.mozilla.com
+    op: auth0
+    url: https://testwebapp.private.mdc1.mozilla.com/callback
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: Hi9rqHLPlnpRfMkc7dtkToFc3DHtcf4Z
+    display: 'false'
+    logo: auth0.png
+    name: rundeck1.private.scl3.mozilla.com
+    op: auth0
+    url: https://rundeck1.private.scl3.mozilla.com/callback
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 3zF88HNSxJLrQqvzz3h1IM3DWTEZ09o4
+    display: 'false'
+    logo: auth0.png
+    name: bz-dev-stats.herokuapp.com
+    op: auth0
+    url: https://bz-dev-stats.herokuapp.com/login
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: hIjZmDmmtkKuaOjmhd5HUzn0ResPDeqX
+    display: 'false'
+    logo: auth0.png
+    name: wde.allizom.org
+    op: auth0
+    url: https://wde-stage.public.mdc1.mozilla.com/redirect_uri
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 7ZuP8yUgV0pQ4siB3paWoLfdq5wRDxCi
+    display: 'false'
+    logo: auth0.png
+    name: firefox-test-tube.herokuapp.com
+    op: auth0
+    url: https://firefox-test-tube.herokuapp.com/accounts/callback/
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 1QcjRY2UEYySI79IqorH94Td590oqXSA
+    display: 'false'
+    logo: auth0.png
+    name: admin.readitlater.localhost
+    op: auth0
+    url: http://admin.readitlater.localhost/auth
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 4u5EiGAICaWFmsyWamVaN1D4f4P6MlR0
+    display: 'false'
+    logo: auth0.png
+    name: shipit.staging.mozilla-releng.net
+    op: auth0
+    url: https://shipit.staging.mozilla-releng.net/login
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 5Ju23B8DLiwzd4bzEHOG2fQHe0R4FRwY
+    display: 'false'
+    logo: auth0.png
+    name: Egencia (US)
```

Comment:
Egencia is strange is that there are multiple "applications" that all use the same client ID, but have specific IDP-initiated URLs, which Egencia uses to detect where to send the user after login. I'd suggest changing the name of this particular entry to something like "Egencia SAML backend" to indicate that this entry is for access control, whereas the other existing entries are for dashboard display and user routing based on groups.